docs/tutorials/podman_pasta_ipv6_with_snid_ipv4.md

```diff
@@ -0,0 +1,761 @@
+        log {
+		output file /var/log/application01.MYDOMAIN.TLD/access.json {
+			roll_size 100MiB
+			roll_keep 5000
+			roll_keep_for 720h
+            roll_uncompressed
+		}
+        format json
+	}
+
+    reverse_proxy http://[::1]:8080
+}
+```
+
+# Redirects
+## General
+When handling Redirects, HTTP Codes `301` (`Moved Permanently`), `302` (`Found (Temporary Redirect)`), `307` (`Temporary Redirect`) and `308` (`Permanent Redirect`) are Typically Used.
+
+References:
+- https://storychief.io/blog/301-302-307-308-redirect
+- https://www.infidigit.com/blog/308-permanent-redirect/
+
+When a 308 redirect code is specified, the client must repeat the exact same request (POST or GET) on the target location. For 301 redirect, the client may not necessarily follow the exact same request. 
+
+Since the HTTPS must always be Enforced (`Permanent Redirect`) and the Client should perform the exact same Request over HTTPS that was performed over HTTP, the Code `308` seems more Appropriate.
+
+## HTTP -> HTTPS for IPv4 Redirects
+The `snid` Service does NOT handle HTTP (non-HTTPS) Requests and does NOT bind to the IPv4 Address on Port 80.
+
+This Problem (IPv4 HTTP -> HTTPS Redirects) can easily be solved by using a one-off (for the entire Podman Host) Caddy Container.
+
+(Option 1) `compose.yml` with "Normal" Ports Mappings:
+```
+services:
+  redirect-http-ipv4-caddy:
+    image: caddy:latest
+    pull_policy: "missing"
+    container_name: redirect-http-ipv4-caddy
+    restart: "unless-stopped"
+    security_opt:
+      - no-new-privileges:true
+      - label=type:container_runtime_t
+    ports:
+      - target: 80
+        host_ip: 172.16.1.10
+        published: 80
+        protocol: tcp
+    network_mode: "pasta:--ipv4-only"
+    volumes:
+      - ./Caddyfile:/etc/caddy/Caddyfile:ro,z
+      - ~/containers/data/redirect-http-ipv4-caddy:/data:rw,z
+      - ~/containers/log/redirect-http-ipv4-caddy:/var/log:rw,z
+      - ~/containers/config/redirect-http-ipv4-caddy:/config:rw,z
+      - ~/containers/certificates/letsencrypt:/certificates:ro,z
+    environment:
+      - CADDY_DOCKER_CADDYFILE_PATH=/etc/caddy/Caddyfile
+
+```
+
+(Option 2) `compose.yml` with "Pasta" One-Liner:
+```
+services:
+  redirect-http-ipv4-caddy:
+    image: caddy:latest
+    pull_policy: "missing"
+    container_name: redirect-http-ipv4-caddy
+    restart: "unless-stopped"
+    security_opt:
+      - no-new-privileges:true
+      - label=type:container_runtime_t
+    network_mode: "pasta:--ipv4-only,-t,172.16.1.10/80"
+    volumes:
+      - ./Caddyfile:/etc/caddy/Caddyfile:ro,z
+      - ~/containers/data/redirect-http-ipv4-caddy:/data:rw,z
+      - ~/containers/log/redirect-http-ipv4-caddy:/var/log:rw,z
+      - ~/containers/config/redirect-http-ipv4-caddy:/config:rw,z
+      - ~/containers/certificates/letsencrypt:/certificates:ro,z
+    environment:
+      - CADDY_DOCKER_CADDYFILE_PATH=/etc/caddy/Caddyfile
+```
```

Comment:
remove

Reply:
I somehow liked your previous idea more (merge the 2, add a comment like `# Pasta One-Liner (DISABLE the Normal Ports Section above)` just below the minimal pasta line, and of course the `network_mode: "pasta:--ipv4-only,-t,172.16.1.10/80"` below commented.

2 Lines overhead instead of a completely copy-pasted compose file. It will behave differently (and the user should be informed about it), but it will cut down the amount of code by a lot.

Would this be acceptable (with a small note explaining the behavioral difference in terms of `podman ps` / `podman inspect`, as already done) ?

Reply:
Merged the two `compose.yml` together.

Excerpt of `compose.yml` showing both possibilities:
```
    # Ports Section (DISABLE if using the Pasta Extended Line)
    ports:
      - target: 80
        host_ip: 172.16.1.10
        published: 80
        protocol: tcp
    # Pasta Minimal Line (ENABLE the Normal Ports Section above if using this Method)
    network_mode: "pasta:--ipv4-only"
    # Pasta Extended Line (DISABLE the Normal Ports Section above if using this Method)
    network_mode: "pasta:--ipv4-only,-t,172.16.1.10/80"
```

Reply:
Closing. Please re-open or open a new Conversation if not satisfactory.

Reply:
I just wonder how much value the extended pasta line has, when the standard approach works. It was necessary while pasta wasn't completely hooked in podman, but now that's the case, there is not much use for it. This is why I suggest to drop it completely.

Reply:
Well depends what you mean by "now that's the case". Not everybody runs the latest Podman Version to begin with. And, again, the two lines behave differently. You can argue that it's only regarding the Reporting, but I find it quite confusing.

But yeah, the "normal Port Section" will show the opened Ports correctly, so you are arguing that it's the "safe approach" (even though you use the one-liner, don't you, you told me you didn't care about Podman reporting the correct info ...) ?